Rubberduck.Parsing/VBA/DeclarationResolveRunnerBase.cs

```diff
@@ -49,8 +49,6 @@ public abstract class DeclarationResolveRunnerBase : IDeclarationResolveRunner
 
         protected void ResolveDeclarations(QualifiedModuleName module, IParseTree tree, CancellationToken token)
         {
-            if (module == null) { return; }
```

Comment:
Why has this been removed? Now, passing in null results in a NRE in the catch block.